CONTIRBUTING.md

```diff
@@ -0,0 +1,129 @@
+# Contributing to _dotenvy_
+
+**Thank you very much for considering to contribute to this project!**
+
+We welcome any form of contribution:
+
+- Asking for help
+- Reporting a bug
+- Adding a feature
+- Fixing a problem
+- Improving the code
+- Sharing ideas
+- Adding documentation
+- Adding examples
+- Fixing spelling/grammar errors
+- _etc..._
+
+**Note**: Before making a large change, it is a good idea to open an issue
+describing the change to solicit feedback and guidance. This will increase the
+likelihood of the PR getting merged and help avoid multiple people writing the
+same solution.
+
+This guide will help you get started. Do not let this guide intimidate you. It
+should be considered a map to help you navigate the process.
+
+## Conduct
+
+The `dotenvy` project adheres to the [Rust Code of
+Conduct](https://www.rust-lang.org/policies/code-of-conduct).
+
+## Issues
+
+Please feel free to open an [issue](https://github.com/allan2/dotenvy/issues),
+or comment on existing issues regarding any topic. If you would like to attempt
+to implement a change, you can ask for help or guidance on solving a problem, or
+just let us know you are working on it.
+
+A maintainer will mark the issue as assigned to you to avoid other people
+unknowingly working on the same thing. If you notice an old assigned issue still
+has no update, please still ask. We can reassign as necessary.
+
+This project sometimes has longer form
+[discussions](https://github.com/allan2/dotenvy/discussions). Feel free to
+comment on existing threads or open your own.
+
+## Pull Requests
+
+Pull Requests are the way concrete changes are made to the code, documentation,
+and dependencies in the dotenvy repository.
+
+Even tiny pull requests (e.g., one character pull request fixing a typo in API
+documentation) are greatly appreciated. Before making a large change, it is
+usually a good idea to first open an issue describing the change to solicit
+feedback and guidance. This will increase the likelihood of the PR getting
+merged.
```

Comment:
```suggestion
Even tiny PRs, like a one character change to fix a typo in the docs, are greatly appreciated. Before making a large change, it is recommended to first open an issue describing the change as to solicit
feedback and guidance. This will increase the likelihood of the PR getting
merged.
```